fastlane/lib/fastlane/actions/update_project_provisioning.rb

```diff
@@ -148,7 +160,8 @@ def self.example_code
             xcodeproj: "Project.xcodeproj",
             profile: "./watch_app_store.mobileprovision", # optional if you use sigh
             target_filter: ".*WatchKit Extension.*", # matches name or type of a target
-            build_configuration: "Release"
+            build_configuration: "Release",
+            codesigning_identity: "iPhone Development" #specified the codesigning identity
```

Comment:
Can you change this to be `# optionally specify the codesigning identity`

Reply:
Thanks for ur advice I had update the code